components/suggest/src/yelp.rs

```diff
@@ -237,20 +237,29 @@ impl SuggestDao<'_> {
             let candidate = candidate_chunk
                 .map(|chunk| chunk.join(" "))
                 .unwrap_or_default();
-            if self.conn.query_row_and_then_cachable(
-                "
-                SELECT EXISTS (
-                    SELECT 1 FROM yelp_modifiers WHERE type = :type AND keyword = :word LIMIT 1
-                )
-                ",
+            if let Ok(keyword_lowercase) = self.conn.query_row_and_then_cachable(
```

Comment:
Let's use `try_query_one` or the other method we talked about it in the other PR, plus `?`.

Reply:
Oh, yes! I forgot it, thanks!